src/lib/components/wizard.component.ts

```diff
@@ -441,4 +482,76 @@ export class WizardComponent implements AfterContentInit, OnChanges {
   public reset(): void {
     this.navigation.reset(this);
   }
+
+  private updateOrientationClasses(): void {
```

Comment:
This method is really hard to understand:
- when is this method called?
- what are its responsibilities?
- what do the internally called methods do?